packages/react-aria-components/docs/Table.mdx

```diff
@@ -196,6 +196,8 @@ import {Table, TableHeader, TableBody, Column, Row, Cell} from 'react-aria-compo
   }
 
   .react-aria-Cell {
+    transform: translateZ(0);
```

Comment:
fixed issue where cells that had any border radius would separate visually from their neighbor